extensions/fairifier-rdf-extension/dtls/fairifier/PostFairDataToFairDataPoint.java

```diff
@@ -137,13 +138,23 @@ public void doPost(HttpServletRequest req, HttpServletResponse res)
             writer.value(distributionPost);
             writer.endObject();
 
-        } catch (Exception ex) {
+        } catch (IOException ex) {
+            respondException(res, ex);
+        } catch (JSONException ex) {
+            respondException(res, ex);
+        } catch (InstantiationException ex) {
+            respondException(res, ex);
+        } catch (MetadataException ex) {
+            respondException(res, ex);
+        } catch (IllegalAccessException ex) {
+            respondException(res, ex);
+        } catch (HttpException ex) {
```

Comment:
How is this (the separate exceptions that are caught) handled in the front-end? Different exceptions have different causes and might require different actions from the user. As it is now, this could also be a multi-catch.